fab/inventory/production

```diff
@@ -60,5 +62,5 @@ datavol_device=/dev/xvdb
 [stanchion]
 hqriak0.internal-va.commcarehq.org
 
-[rackspace_cloud_backup]
-hqdb0.internal.commcarehq.org
+# [rackspace_cloud_backup]
+# hqdb0.internal-va.commcarehq.org
```

Comment:
This is a backup of the `/opt/data` dir on that machine.  I think we need to revisit this approach anyways, I think @calellowitz was doing some work on that front.  I just sent an email to follow up.